.github/workflows/go.yaml

```diff
@@ -0,0 +1,23 @@
+name: Go
+
+on: push
```

Comment:
Also, please make it run on every PR or on master branch

Reply:
I change it to run on master and P